src/ray/raylet/scheduling/cluster_resource_scheduler_test.cc

```diff
@@ -574,7 +574,9 @@ TEST_F(ClusterResourceSchedulerTest, GetLocalAvailableResourcesWithCpuUnitTest)
   ClusterResourceScheduler resource_scheduler(0, node_resources, *gcs_client_);
 
   TaskResourceInstances available_cluster_resources =
-      resource_scheduler.GetLocalResources().GetAvailableResourceInstances();
+      resource_scheduler.GetLocalResourceManager()
```

Comment:
We are planning to not include local resource manager within cluster resource manager in the future right?

Reply:
yes